documentation/build.adoc

```diff
@@ -0,0 +1,12 @@
+:toc:
```

Comment:
As discussed in our meetings we do not want to come back to having help documentation on github in asciidoc.
This has many disadvantages (not available offline, esp. not accurate if the installed release differs from the latest documentation in development, no localization, not intuitive for CLI users, etc.).
Simply use your invest to put this into the CLI help output. Also please add the new variables to the `variables.adoc`. Then you can remove these 5 added adocs.

Reply:
For this we already agreed in the team to add yet another help property for the detailed description of a commandlet.
So `ide help «commandlet»` will not only print `cmd.build` but also `cmd.build.detail`.
I can prepare the PR for this change.

Reply:
Thanks, I've removed the documentation now and added a detail for build in English and German (translated with Copilot)